Makefile

```diff
@@ -539,6 +539,21 @@ endif
 unpack_dist:
 	@tar -xf $(THEMIS_DIST_FILENAME)
 
+nsis_installer: $(BIN_PATH)/InstallThemis.exe
+
+$(BIN_PATH)/InstallThemis.exe: FORCE
+	@$(MAKE) install PREFIX=/ DESTDIR="$(BIN_PATH)/install"
+# Windows users will need MSYS runtime libraries, so copy dependencies
+# into the installation directory as well
+ifdef IS_MSYS
+	@ldd "$(BIN_PATH)/install/bin"/*.dll | \
+	 awk '$$3 ~ "^/usr/bin" { print $$3}' | sort --uniq | \
+	 xargs -I % cp % "$(BIN_PATH)/install/bin"
+endif
+	@makensis Themis.nsi
+	@rm -r "$(BIN_PATH)/install"
+
+FORCE:
```

Comment:
what means `FORCE`?

Reply:
This empty target forces the installer to be rebuilt when `make nsis_installer` is run. Otherwise make sees that `InstallThemis.exe` already exists in the build directory and does not need to be updated.

Ideally, we should not be calling `make install` recursively. The **nsi** script could copy the binaries from the build directory and header files from the repo. This will allow us to accurately track the dependencies for make so that repeated `make nsis_installer` runs will rebuild the binaries and the installer only when necessary.

However, that would mean that we'd have to maintain two separate lists of installed files. Recently we have unified RPM and DEB packaging to use `make install` instead of their own lists. I think that we'd be better off just using `make install` everywhere.